test/fixtures/simple/quoted_pattern.mf2

```diff
@@ -7,6 +7,10 @@ Text                ^^^^^^^^^^^^^^^ 0:0-0:15
 Quoted pattern is not allowed inside of a pattern. (at @6..8)
   Hello {{World}}
         ^^
+=== fixed ===
+Remove quotes:
```

Comment:
We should also have a quickfix to replace `{{` with `{`.

Reply:
Idk - that seems strange. To make this workable we'd only have to surface the quickfix only if the pattern in the quoted pattern parses as an expression.